src/components/MagicCodeInput.js

```diff
@@ -300,10 +300,7 @@ function MagicCodeInput(props) {
         }
     };
 
-    // We need to check the browser because, in iOS Safari, an input in a container with its opacity set to
-    // 0 (completely transparent) cannot handle user interaction, hence the Paste option is never shown.
-    // Alternate styling will be applied based on this condition.
```

Comment:
The comment is still true I think? But just needs to be updated to tell the full story?